docs/docs-components/siteIndex.js

```diff
@@ -175,6 +175,10 @@ const siteIndex: $ReadOnlyArray<siteIndexType> = [
         sectionName: 'Color',
         pages: ['Palette', 'Usage', 'Examples'],
       },
+      {
+        sectionName: 'Content',
+        pages: ['Guidelines'],
```

Comment:
Are there plans for more pages under the "Content" umbrella?

Reply:
Good question. Just asked it in our chat! I'm not exactly sure